File: official/docs/java/users/DeleteChildUser.java
import com.easypost.EasyPost;

public class DeleteChildUser {
    public static void main(String[] args) {
        EasyPost.apiKey = "<EASYPOST_API_KEY>";

        User user = User.retrieve("user_...");

        user.delete();

        System.out.println(user);
    }
}

File: official/docs/csharp/users/delete-child-user.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task DeleteChildUser()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        User user = await User.Retrieve("user_...");

        bool success = await user.Delete();

        new TestOutputHelper().WriteLine(success.ToString());
    }
}
